tests/foreman/ui/test_computeresource_gce.py

```diff
@@ -28,33 +28,10 @@
 from robottelo.helpers import download_gce_cert
 from robottelo.test import settings
 
-pytestmark = [pytest.mark.skip_if_not_set('gce')]
```

Comment:
Cleaned up the fixture use in this module since I was moving some marks around anyway.